pkg/limits/frontend/frontend.go

```diff
@@ -243,6 +244,10 @@ func (f *Frontend) ExceedsLimits(ctx context.Context, req *logproto.ExceedsLimit
 	}, nil
 }
 
+func (f *Frontend) GetStreamUsageGatherer() *RingStreamUsageGatherer {
+	return f.streamUsage.(*RingStreamUsageGatherer)
```

Comment:
I don't think we should be doing this, is this just for the HTTP handler?

Reply:
Yes exactly it is only about to have the escape hatch HTTP handler to invalidate the cache manually. I want to remove this and move this static html handler to our operational UI in a follow up PR.

Reply:
Will remove this with used the separate handler methods in modules.go